src/Compilers/CSharp/Portable/BoundTree/Constructors.cs

```diff
@@ -203,6 +203,7 @@ public static BoundConversion SynthesizedNonUserDefined(SyntaxNode syntax, Bound
                 @checked: false,
                 explicitCastInCode: false,
                 constantValueOpt: constantValueOpt,
+                isNullable: false,
```

Comment:
Should we use oblivious/null for synthesized nodes, to indicate we don't really care?

Reply:
Yes. Changed.